### ISSUE TITLE
2015-08-11 CaaC Standup

### ISSUE THREAD

**Issue by MatthewVita** [saxifrage/learn#170]
previous standup: https://github.com/saxifrage/learn/issues/162

What I accomplished last week:
- Implemented latest tweaks to the Explore Cards for https://github.com/saxifrage/learn/issues/93 (this is almost done... just need to tweak a few more things per @chelseaerdner's helpful comments)
- @Tim-Stimach and I found that https://github.com/saxifrage/learn/issues/152 was already done so we verified it by adding better seed data
- Caught up on the state of the admin piece and asked Justin to prefix the admin area with `/admin` here: https://github.com/saxifrage/learn/issues/165
- Universal Badge API idea discussion made good progress... we've narrowed our ideal API to https://github.com/saxifrage/universal-badge-api/blob/master/README.md -- perhaps, we can reach out the Badge Alliance to get their thoughts?

What I intend to do this week:
- Finish last tweaks to the Explore Cards for https://github.com/saxifrage/learn/issues/93
- Continue to think about Universal Badge API
- Finish off CaaC Guidelines

Obstacles that are impeding my progress:
- Waiting on @Arf375's vote so we can continue with CaaC Guidelines https://github.com/saxifrage/learn/issues/151

**Comment by MatthewVita** [comment]
![img](https://upload.wikimedia.org/wikipedia/commons/thumb/5/55/Arecibo_message.svg/320px-Arecibo_message.svg.png)

A graphical representation of the Arecibo message, Humanity's first attempt to use radio waves to actively communicate ~~its existence to alien civilizations~~ a reminder for the CaaC members to submit their standup.

#IWantToBelieve

**Comment by MatthewVita** [comment]
dat Fermi Paradox

**Comment by chadwhitacre** [comment]
What I accomplished last week:
- Vacationed. :)

What I intend to do this week:
- Work on multi-maps (https://github.com/saxifrage/caac-map/issues/11) and pathways (https://github.com/saxifrage/caac-map/issues/19) on Friday.

Obstacles that are impeding my progress:
- None.

**Comment by chadwhitacre** [comment]
> [Ever wondered](https://twitter.com/hairyfotr/status/403241219444076544) what happens, when you put the Arecibo message into the Conway's game of life?

Why yes, yes I have. :-)

Sooooo I wrote a [script](https://gist.github.com/whit537/1003a0a83e1e89f95999) to generate an URL for [this](http://pmav.eu/stuff/javascript-game-of-life-v3.1.1/) Life implementation, using the binary representation [available](https://en.wikipedia.org/wiki/Arecibo_message#Message_as_binary_string) at Wikipedia. :-)

[![Arecibo of Life](https://cloud.githubusercontent.com/assets/134455/9223161/9a8a7b90-40c5-11e5-9397-868231cad82b.png)](http://pmav.eu/stuff/javascript-game-of-life-v3.1.1/?s=[{%2256%22:[78,80,81,82,87,88,89,95,96,98,99,100],%2277%22:[85,88,90],%2254%22:[80,85,87,88,89,91,94,96,97,99,100],%2242%22:[84,85,89,94,95],%2248%22:[80,88,96],%2243%22:[82,83,88,89,96],%2260%22:[80,86,87,89,90],%2261%22:[],%2262%22:[80,81,82,88],%2263%22:[80,81,82,84,86,90,92,94,96,98,100],%2264%22:[80,81,82,92,94,96,98],%2249%22:[81,94,95],%2266%22:[86,87,88,89,90],%2267%22:[84,85,86,87,88,89,90,91,92],%2268%22:[82,83,84,92,93,94],%2269%22:[81,82,94,95],%2252%22:[80,88],%2253%22:[80,86,87,88,89,90],%2224%22:[82,100],%2225%22:[78,79,80,81,82,96,97,98,99,100],%2226%22:[],%2227%22:[78,79,84,85,90,91,92,96,97],%2220%22:[78,79,80,81,82,84,85,86,87,88,90,91,92,93,94,96,97,98,99,100],%2221%22:[],%2222%22:[81,99],%2223%22:[],%2246%22:[79,88,89,98],%2247%22:[79,89,97],%2244%22:[81,88,97],%2245%22:[80,88,89,98],%2228%22:[78,86,96],%2229%22:[78,79,81,86,87,91,92,93,96,97,99],%2240%22:[84,85,89,94,95],%2241%22:[88,89,92,93],%227%22:[80,82,88,90,98],%226%22:[84,86,88,90],%229%22:[78,80,82,84,86,88,90,92,95,98],%228%22:[78,82,86,90,93,95,96,99],%2278%22:[79,80,81,82,85,86,87,88,89,91,94,95,96,97],%2251%22:[80,84,85,86,88,90,91],%2239%22:[82,83,88,89,96],%2265%22:[92,94],%2276%22:[83,93],%2275%22:[83,88,93],%2238%22:[81,88,89,97],%2273%22:[79,83,86,90,93,97],%2272%22:[79,83,85,91,93,97],%2271%22:[79,80,83,84,92,93,96,97],%2270%22:[80,81,83,93,95,96],%2259%22:[80,87,89,95,96],%2258%22:[80,87,89,95,96,97,98,99,100],%2211%22:[90,91],%2210%22:[],%2213%22:[88,89,91],%2212%22:[88,89,91],%2215%22:[87,88,89,90,91],%2214%22:[87,89,91],%2217%22:[78,79,84,85,86,90,91,96,97],%2216%22:[],%2219%22:[78,79,81,85,86,90,91,96,97,99],%2218%22:[78,92,93,96],%2231%22:[],%2230%22:[78,79,80,81,82,84,85,86,87,88,90,91,92,93,94,96,97,98,99,100],%2237%22:[80,89,98],%2236%22:[88,89],%2235%22:[78,79,80,81,82,88,89,96,97,98,99,100],%2234%22:[82,88,89,100],%2233%22:[88,89],%2255%22:[84,87,88,89,92,95,96,97,98,99,100],%2274%22:[83,87,89,93],%2232%22:[81,88,89,99],%2257%22:[87,89,95,96,97,99,100],%2250%22:[82,83,92,93]}])

http://tinyurl.com/p3ymn8v

**Comment by MatthewVita** [comment]
@whit537 assuming the message never made it to any life forms after it was sent, you've quite literally allowed the most non-human lifeforms to interact on the Arecibo message to date.

**Comment by chadwhitacre** [comment]
:alien:
